### DISCUSSION TITLE
Best script to run to get fleet-server back?

### DISCUSSION THREAD

**Discussion by jaypfe** [Security-Onion-Solutions/securityonion#12520] [Unsupported Versions]
Hey guys! 

a little while ago I had an issue my dedicated fleet node showing 'inactive' in fleet. The node is up in the general policy, but the fleet server piece shows inactive.  #11530 (this issue was towards the bottom) 

I was able to fix it with so-elastic-fleet-restart --force. Since then however I see in the release notes I have seen the inclusion of some new scripts related to fleet. 

I was just curious if there was a new script I should run to bring the fleet server back to active status or if the best method is still so-elastic-fleet-restart --force

Thanks again!

**Comment by cm-ops** [comment]
If you restart fleet with `so-elastic-fleet-restart`, does it come up and is it working as expected?

**Comment by jaypfe** [comment]
Yup, that did it. 

Thanks again!
